### DISCUSSION TITLE
Search results (Find results) in form of an interactive list of all found occurences

### DISCUSSION THREAD

**Discussion by Diabl0pl** [sumatrapdfreader/sumatrapdf#3567] [Q&A]
Hello,

Is it possible to display search results in form of an interactive list displaying all occurences of a searched phrase so that we could easy jump between found positions? 

Image for reference:

![search resultspng](https://github.com/sumatrapdfreader/sumatrapdf/assets/8557443/9dba3505-1fc2-4c2f-99eb-d286021c89e5)

**Comment by GitHubRulesOK** [comment]
SumatraPDF does not have such a feature , it simply quickly looks a head to the next separate character (No such thing as a word object in PDF language)

But using heavier PDF viewers could help look more slowly OR use a fast indexer like MythicSoft AgentRansack which can keep a copy of all the characters for multiple look-up (Ensure it is a powerful one like Acrobat that stores ALL the file contents for searching)
